src/proposals/20230830/SparkEthereum_20230830.t.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import '../../SparkTestBase.sol';
+
+import { Ownable } from 'aave-v3-core/contracts/dependencies/openzeppelin/contracts/Ownable.sol';
+
+import { SparkEthereum_20230830 } from './SparkEthereum_20230830.sol';
+
+contract SparkEthereum_20230830Test is SparkEthereumTestBase {
+
+    address public   constant WETH                           = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    uint256 public   constant OLD_WETH_OPTIMAL_USAGE_RATIO   = 0.80e27;
+    uint256 public   constant NEW_WETH_OPTIMAL_USAGE_RATIO   = 0.90e27;
+    uint256 public   constant OLD_WETH_VARIABLE_RATE_SLOPE_1 = 0.030e27;
+    uint256 public   constant NEW_WETH_VARIABLE_RATE_SLOPE_1 = 0.028e27;
+    uint256 public   constant OLD_WETH_VARIABLE_RATE_SLOPE_2 = 0.80e27;
+    uint256 public   constant NEW_WETH_VARIABLE_RATE_SLOPE_2 = 1.20e27;
+
+    address public   constant wstETH                         = 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0;
+    uint256 public   constant OLD_WSTETH_SUPPLY_CAP          = 200_000;
+    uint256 public   constant NEW_WSTETH_SUPPLY_CAP          = 400_000;
+
+    constructor() {
+        id = '20230830';
+    }
+
+    function setUp() public {
+        vm.createSelectFork(getChain('mainnet').rpcUrl, 17_935_850);
```

Comment:
Lets comment this out until the spell is deployed

Reply:
Not sure what you mean. Can you elaborate?

Reply:
He means it's good to remove pinning to a specific block until the spell is deployed.

Reply:
Ok, got it